module/zfs/spa_misc.c

```diff
@@ -3133,6 +3145,9 @@ ZFS_MODULE_PARAM(zfs, zfs_, flags, UINT, ZMOD_RW,
 ZFS_MODULE_PARAM(zfs, zfs_, recover, INT, ZMOD_RW,
 	"Set to attempt to recover from fatal errors");
 
+ZFS_MODULE_PARAM(zfs, zfs_, recover_ms, INT, ZMOD_RW,
+	"Set to attempt to recover from fatal errors during metaslab loading");
```

Comment:
I am not sure it is really about metaslab loading, or even about metaslabs.  Doesn't it apply to all spacemap operations, or even some non-spacemap?  Also as I have mentioned during the call, I recall it more often happening not even during the metaslab loading (under which I guess you mean spacemap condensing, or whatever it is called), but when deleting snapshots, moving their list of blocks to parent of to free space.